install-script/install.sh

```diff
@@ -131,6 +137,10 @@ CURL_MAX_TIME=1800
 # set by check_dependencies therefore cannot be set by set_defaults
 SYSTEMD_DISABLED=false
 
+DISABLE_INSTALLATION_TELEMETRY=false
+INSTALLATION_LOGFILE="/tmp/sumologic-otel-collector_installation.log"
```

Comment:
You should use `${TMPDIR}` as the base instead. That has a temp directory with the right permissions on all supported systems.

Reply:
100% agree ; good catch